cli/tools/bench/mod.rs

```diff
@@ -180,6 +214,10 @@ async fn bench_specifier(
   worker.execute_side_module_possibly_with_npm().await?;
 
   let mut worker = worker.into_main_worker();
+
+  // Ensure that there are no pending exceptions before we start running tests
+  worker.run_up_to_duration(Duration::from_millis(0)).await?;
```

Comment:
Isn't this racy? Ie. won't in some situations timeouts fire before event loop tick finishes?